### ISSUE TITLE
The installation of molsimplify via pip is missing necessary data files.

### ISSUE THREAD

**Issue by HelloJocelynLu** [hjkgrp/molSimplify#243]
Hello everyone,

I am currently trying to integrate molsimplify into my project. I followed the instructions in the README file to install molsimplify using pip. However, when I attempted to add a ligand using the command:

```
molsimplify -ligadd temp_mol.xyz -ligname RUCBEY-subgraph-1 -ligcon 3 -skipANN True
```

I encountered the following errors:
```
FileNotFoundError: [Errno 2] No such file or directory: '/xxxxx/molSimplify/Cores/cores.dict'
FileNotFoundError: [Errno 2] No such file or directory: '/xxxxx/molSimplify/Data/coordinations.dict'
FileNotFoundError: [Errno 2] No such file or directory: '/xxxxx/molSimplify/Bind/bind.dict'
FileNotFoundError: [Errno 2] No such file or directory: '/xxxxx/molSimplify/Substrates'
```

I resolved the issue by manually copying the relevant folders and files from the repository.

It might be necessary to update the toml file to ensure all required data files are included for pip installation?

https://github.com/hjkgrp/molSimplify/blob/main/pyproject.toml#L43-L50

Thank you,
Jocelyn

**Comment by gianmarco-terrones** [comment]
Hi Jocelyn,

Did you install with the README instructions in section **via pip, from PyPI**, or with **via pip, from GitHub**? I assume the former?

**Comment by gianmarco-terrones** [comment]
Also, in case this helps, another way to add ligands is to add geometry (e.g., XYZ) files to the folder `molSimplify/Ligands`, and to add lines to `ligands.dict` in that same folder.

**Comment by gianmarco-terrones** [comment]
Anyway, thanks for indicating where changes should be made. I have made changes in the `pyproject.toml` file. We have things rigged up such that a PyPi release occurs [every time a new version is released](https://github.com/hjkgrp/molSimplify/blob/main/.github/workflows/build.yaml), so this change won't fix things in the 1.7.4 PyPi release, but it should prevent the issue for any future releases.

**Comment by HelloJocelynLu** [comment]
> Hi Jocelyn,
> 
> Did you install with the README instructions in section **via pip, from PyPI**, or with **via pip, from GitHub**? I assume the former?

I installed **via pip, from PyPI**


> Anyway, thanks for indicating where changes should be made. I have made changes in the `pyproject.toml` file. We have things rigged up such that a PyPi release occurs [every time a new version is released](https://github.com/hjkgrp/molSimplify/blob/main/.github/workflows/build.yaml), so this change won't fix things in the 1.7.4 PyPi release, but it should prevent the issue for any future releases.

Thank you for the update!

> Also, in case this helps, another way to add ligands is to add geometry (e.g., XYZ) files to the folder `molSimplify/Ligands`, and to add lines to `ligands.dict` in that same folder.

Thank you for letting me know!
